File: GYM.Controller.Api/DTOs/StatsDTO.cs
namespace GYM.Controller.Api.DTOs;
public class StatsDTO
{
    public int Id {get; set;}
    public int UserId{get; set;}
    public decimal Weight {get;set;} 
    public decimal Height {get;set;}
    public decimal Strength {get;set;}
    public TimeOnly MileRun{get;set;}
    public DateOnly MeasureAt{get;set;} 
    public int Age {get; set;}
}

File: GYM.FrontEnd/src/types/StatsDTO.ts
// Representa un registro individual de medición física/rendimiento
export interface StatsDTO {
  id: number;
  userId: number;
  weight: number;
  height: number;
  strength: number;
  mileRun: string;
  measureAt: string;
  age: number;
}

// create stadistics
export type CreateStatisticBody = Omit<StatsDTO, "id" | "measureAt" | "userId">; // omitting id, measureAt, and userId because backend sets it from auth